sql/importer: TestExportImportBank failed

sql/importer.TestExportImportBank [failed](https://tanzanite.cluster.engflow.com/invocations/default/79c893aa-5ebc-4c4c-aa17-3b0e4cc0d55c?testReportRun=3&testReportShard=3&testReportAttempt=1#targets-Ly9wa2cvc3FsL2ltcG9ydGVyOmltcG9ydGVyX3Rlc3Q=) on master @ [b77653f98b4ce56425f7b0997671d8414051ea54](https://github.com/cockroachdb/cockroach/commits/b77653f98b4ce56425f7b0997671d8414051ea54):


```
=== RUN   TestExportImportBank
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestExportImportBank80965495
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:152: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    exportcsv_test.go:141: -- test log scope end --
test logs left over in: outputs.zip/logTestExportImportBank80965495
--- FAIL: TestExportImportBank (476.66s)
=== RUN   TestExportImportBank/null=NULL
    exportcsv_test.go:130: error executing query="IMPORT INTO bank2 CSV DATA ($1) WITH delimiter = '|', nullif = 'NULL'" args=["userfile:/t/TestExportImportBank/null=NULL/*"]: pq: job 1024450313194471425: could not mark as reverting: job 1024450313194471425: with status "running": expected session "0101801d87ef14c2d542d0afe588074c706a05" but found "010180f79889bac6484f23a01397e43091f392"
    panic.go:626: error executing query="DROP TABLE bank2" args=[]: pq: relation "bank2" is offline: importing
    --- FAIL: TestExportImportBank/null=NULL (180.30s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=3</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestExportImportBank.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-44957

--- a/pkg/sql/sqlliveness/slinstance/BUILD.bazel
+++ b/pkg/sql/sqlliveness/slinstance/BUILD.bazel
@@ -12,6 +12,7 @@ go_library(
         "//pkg/sql/sqlliveness",
         "//pkg/sql/sqlliveness/slbase",
         "//pkg/sql/sqlliveness/slstorage",
+        "//pkg/util",
         "//pkg/util/grpcutil",
         "//pkg/util/hlc",
         "//pkg/util/log",

--- a/pkg/sql/sqlliveness/slinstance/slinstance.go
+++ b/pkg/sql/sqlliveness/slinstance/slinstance.go
@@ -20,6 +20,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/sqlliveness"
 	"github.com/cockroachdb/cockroach/pkg/sql/sqlliveness/slbase"
 	"github.com/cockroachdb/cockroach/pkg/sql/sqlliveness/slstorage"
+	"github.com/cockroachdb/cockroach/pkg/util"
 	"github.com/cockroachdb/cockroach/pkg/util/grpcutil"
 	"github.com/cockroachdb/cockroach/pkg/util/hlc"
 	"github.com/cockroachdb/cockroach/pkg/util/log"
@@ -415,7 +416,11 @@ func NewSQLInstance(
 		stopper:        stopper,
 		sessionEvents:  sessionEvents,
 		ttl: func() time.Duration {
-			return slbase.DefaultTTL.Get(&settings.SV)
+			ttl := slbase.DefaultTTL.Get(&settings.SV)
+			if util.RaceEnabled {
+				ttl *= 5
+			}
+			return ttl
 		},
 		hb: func() time.Duration {
 			return slbase.DefaultHeartBeat.Get(&settings.SV)

